osbrain/tests/test_agent.py

```diff
@@ -644,16 +658,16 @@ def test_agent_execute_as_function(nsproxy):
     in the remote agent.
     """
     class EnvironmentAgent(Agent):
-        def configure(self, value):
+        def set_up(self, value):
             os.environ['__OSBRAIN_TEST'] = value
 
     def name(prefix, suffix='suffix'):
         return prefix + os.environ.get('__OSBRAIN_TEST', '') + suffix
 
     agent0 = run_agent('a0', base=EnvironmentAgent)
     agent1 = run_agent('a1', base=EnvironmentAgent)
-    agent0.configure('0')
-    agent1.configure('1')
+    agent0.set_up('0')
+    agent1.set_up('1')
```

Comment:
Not very important. For me `.set_environment()` would sound better than `.set_up()`. If you don't agree, leave it as-is.

Reply:
Didn't look too much into this test. I just used the first similar word to "configure" I could come up with 😄